web/pluginToStandaloneMapper/mock/mock.ts

```
@@ -0,0 +1,228 @@
+                  served: true,
+                  storage: true,
+                  schema: {
+                    openAPIV3Schema: flowMetricSchema
+                  }
+                }]
+              }
+            });
+          }
+          break;
+        case 'FlowCollector':
+          const fc = _.cloneDeep(getFlowCollectorJS());
```

Biome (2.4.15)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)